core/src/test/java/apoc/meta/MetaTest.java

```diff
@@ -1089,4 +1089,45 @@ public void testMetaStatsWithTwoDots() {
         });
 
     }
+
+    @Test
+    public void testMetaStatsWithLabelAndRelTypeCountInUse() {
+        db.executeTransactionally("CREATE (:Node:Test)-[:REL {a: 'b'}]->(:Node {c: 'd'})<-[:REL]-(:Node:Test)");
+        db.executeTransactionally("CREATE (:A {e: 'f'})-[:ANOTHER {g: 'h'}]->(:C)");
+        
+        TestUtil.testCall(db, "CALL apoc.meta.stats()", row -> {
+            assertEquals(map("A", 1L, "C", 1L, "Test", 2L, "Node", 3L), row.get("labels"));
+            assertEquals(5L, row.get("nodeCount"));
+            assertEquals(4L, row.get("labelCount"));
+            
+            assertEquals(map("REL", 4L, "ANOTHER", 1L), row.get("relTypesCount"));
```

Comment:
```suggestion
            assertEquals(map("REL", 2L, "ANOTHER", 1L), row.get("relTypesCount"));
```
shouldn't it be 2?

Reply:
The 4 is due to double label nodes `:Node:Test`.
In this case, the `apoc.meta.stats` count separately 2 relationship `(:Test)-[:REL]-(:Node)` and 2 `(:Node)-[:REL]-(:Node)`.
Instead, using a single labels, `REL` count in 2.

Reply:
Could we add something in the docs for this to mention that it counts unique patterns where multi-label nodes will be counted as separate paths?

Reply:
Of course, I added the test case example in usage doc page